api/src/main/java/org/semanticweb/owlapi/io/RDFParserMetaData.java

```diff
@@ -39,21 +39,22 @@
     private final RDFOntologyHeaderStatus headerStatus;
     private final Set<RDFTriple> unparsedTriples;
     private final ArrayListMultimap<IRI, Class<?>> guessedDeclarations;
-
+    private final Set<IRI> rdfsClassDeclarations;
     /**
      * @param headerStatus
      *        the header status
      * @param tripleCount
      *        the triple count
      * @param unparsedTriples
-     *        the set of triples not parsed
+ *        the set of triples not parsed
      * @param guessedDeclarations
-     *        guessed declarations map
+     * @param rdfsClassDeclarations
      */
     public RDFParserMetaData(@Nonnull RDFOntologyHeaderStatus headerStatus,
-            int tripleCount, @Nonnull Set<RDFTriple> unparsedTriples,
-            @Nonnull ArrayListMultimap<IRI, Class<?>> guessedDeclarations) {
+                             int tripleCount, @Nonnull Set<RDFTriple> unparsedTriples,
+                             @Nonnull ArrayListMultimap<IRI, Class<?>> guessedDeclarations, Set<IRI> rdfsClassDeclarations) {
```

Comment:
Definitely. Also, I hate having all this stuff in a class that is part of a public API.